pkg/zero/connect/connect.proto

```diff
@@ -32,6 +33,30 @@ message ConfigUpdated {
 // config.
 message BootstrapConfigUpdated {}
 
+// TelemetryRequest is sent to request current telemetry data from the Pomerium Core to be sent to the Zero Cloud.
+message TelemetryRequest {
+  // include_session_analytics requests current MAU/DAU data
+  optional SessionAnalyticsRequest session_analytics = 1;
+  // envoy_metrics requests current envoy metrics
+  optional EnvoyMetricsRequest envoy_metrics = 2;
+  // pomerium_metrics requests current pomerium metrics
+  optional PomeriumMetricsRequest pomerium_metrics = 3;
+}
+
+// SessionAnalyticsRequest is used to request current MAU/DAU data
+message SessionAnalyticsRequest {}
+
+// EnvoyMetricsRequest is used to request current envoy metrics
+message EnvoyMetricsRequest {
+  // only include metrics that match the provided labels, and relabel them with the provided labels
+  map<string, string> metrics = 1;
+  // only include labels that match the provided labels
+  map<string, string> labels = 2;
```

Comment:
Could you expand on these comments? I'm not sure I understand what they're saying.

In particular, what do the values in the `labels` map represent?

Reply:
the second value is a name of a metric to be assigned. however, relabeling may be a responsibility of the telemetry receiver so it may be just `repeated string`.

Reply:
Sure, `repeated string` sounds fine to me.